### ISSUE TITLE
responseBodyTests issue when value = null

### ISSUE THREAD

**Issue by thim81** [apideck-libraries/portman#414]
### Discussed in https://github.com/apideck-libraries/portman/discussions/412

<div type='discussions-op-text'>

<sup>Originally posted by **jopucpla** October  5, 2022</sup>
Hello! :) 

I'm currently facing an issue when trying to validate a content on an specific request (using variations).

I added following contentTests on responseBody

```
          contentTests:
            - responseBodyTests:
                - key: proposals[0].supplier.name
                  value: AAA
                - key: proposals[0].supplier.legalIdentifier
                  value: 123 456 789
                - key: proposals[0].supplier.externalId
                  value: 196893 
```

This is converted into the pm assertion below:

```
// Response body should have value "196893" for "supplier.externalId"
if (jsonData?.supplier?.externalId) {
pm.test("[GET]::/proposals/:proposalId - Content check if value for 'supplier.externalId' matches '196893'", function() {
  pm.expect(jsonData.supplier.externalId).to.eql("196893");
})};
```

The problem is when jsonData.supplier.externalId is = null this condition is not met and the validation is skipped.

for example:

    "supplier": {
        "name": "AAA",
        "legalIdentifier": "123 456 789",
        "externalId": null,
        "seller": {
            "firstName": "sa",
            "lastName": "ta",
            "email": "XXX"
        },
        "consultant": {
            "firstName": "XX",
            "lastName": "XX"
        }
    },

![image](https://user-images.githubusercontent.com/60114539/194055016-503858a5-3e4e-4d90-b2ec-cea4ec77dcd5.png)

Based on how the API works the field is nullable, therefore I can't force as not nullable by schema and I want specific request to have this field filled.

Did anyone had similar issues? Is there a way to do this?
I tried with assertions but still same issue since the if clause is still injected and therefore the test bypassed, so the **assert: not.to.be.null** is actually not working at all. Am I doing anything wrong?

Thank you!</div>

**Comment by jopucpla** [comment]
thanks!

**Comment by thim81** [comment]
@jopucpla We have a PR ready, so in the next release, this issue wil be solved.

**Comment by thim81** [comment]
@jopucpla The improved `null` support is added for contentTests & assignVariables. It is included in the Portman 1.19 release.
